common/templates/includes/svg-icons.njk

```diff
@@ -0,0 +1,17 @@
+<div class="is-hidden">
```

Comment:
As mentioned when we chatted this feels nice in terms of flexibility in potentially adding more icons but it also feels poor in terms of performance to have this loaded on every page. Particularly right now where we are only using these icons on ~20% of the pages we have.

I wonder if it this block could be added to the icon component so that at least it's only ever loaded on pages where an icon has been included?

Reply:
> I wonder if it this block could be added to the icon component so that at least it's only ever loaded on pages where an icon has been included?

multiple icons on a page though would mean multiples of the `SVGs`. I don't feel it's too bad size wise and as we are not going to have many icons then file size wise it shouldn't be a problem. At the moment these 2 icons are adding `0.2kb` to the html

## Without inline svg
![without-inline-svg](https://user-images.githubusercontent.com/2305016/65617936-69495900-dfb5-11e9-8059-771ca8eb1220.png)

## With inline svg
![with-inline-svg](https://user-images.githubusercontent.com/2305016/65617935-68b0c280-dfb5-11e9-9677-3f4ec4d8179a.png)